flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -1114,7 +1114,8 @@ class JobManager(
           Option(jobGraph.getSerializedExecutionConfig()
```

Comment:
True, you're right because it would prevent the `StreamingJobGraphGenerator` from setting the `FixedDelayRestartStrategy` in case that no other restarting strategy has been set.